templates/default/apache2.conf.erb

```diff
@@ -18,6 +18,7 @@
   Alias /stylesheets /etc/<%= node['nagios']['server']['vname'] %>/stylesheets
 <% end -%>
   ScriptAlias <%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
+  ScriptAlias /cgi-bin/statusjson.cgi <%= node['nagios']['cgi-bin'] %>/statusjson.cgi
```

Comment:
I would have to check the cookbook and try it. I believe I chose to hard
code it because the JavaScript call was hard coded. It seemed like if the
variable was changed the JavaScript call would fail again, right?

On Tuesday, March 17, 2015, Sander Botman <notifications@github.com> wrote:

> In templates/default/apache2.conf.erb
> <https://github.com/tas50/nagios/pull/343#discussion_r26570137>:
>
> > @@ -18,6 +18,7 @@
> >    Alias /stylesheets /etc/<%= node['nagios']['server']['vname'] %>/stylesheets
> >  <% end -%>
> >    ScriptAlias <%= node['nagios']['cgi-path'] %> <%= node['nagios']['cgi-bin'] %>
> > +  ScriptAlias /cgi-bin/statusjson.cgi <%= node['nagios']['cgi-bin'] %>/statusjson.cgi
>
> Would it not be better to do:
>
> ScriptAlias <%= node['nagios']['cgi-path'] %>/statusjson.cgi <%= node['nagios']['cgi-bin'] %>/statusjson.cgi
>
> Because it feels like hardcoding /cgi-bin is not very handy, but I can be
> wrong here....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tas50/nagios/pull/343/files#r26570137>.
>